packages/@aws-cdk/aws-lambda/lib/image-function.ts

```diff
@@ -41,8 +43,12 @@ export abstract class DockerImageCode {
    */
   public static fromImageAsset(directory: string, props: AssetImageCodeProps = {}): DockerImageCode {
     return {
-      _bind() {
-        return new AssetImageCode(directory, props);
+      _bind(architecture?: Architecture) {
+        return new AssetImageCode(directory, {
+           // determine the platform from `architecture`.
```

Comment:
```suggestion
          // determine the platform from `architecture`.
```
Oops. My suggestion's indentation did you dirty.